### ISSUE TITLE
Unable to install ShortCodes.jl

### ISSUE THREAD

**Issue by VivaldoMendes** [hellemo/ShortCodes.jl#6]
Wonderful stuff. But I tried to install the most recent version on Julia 1.5.4 and got this error:

```
(@v1.5) pkg> add https://github.com/hellemo/ShortCodes.jl
    Cloning git-repo `https://github.com/hellemo/ShortCodes.jl`
   Updating git-repo `https://github.com/hellemo/ShortCodes.jl`
  Resolving package versions...
ERROR: Unsatisfiable requirements detected for package Plotly [58dd65bb]:
 Plotly [58dd65bb] log:
 ├─possible versions are: [0.2.0, 0.3.0] or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions [0.2.0, 0.3.0]
 └─restricted by compatibility requirements with HTTP [cd3eb016] to versions: uninstalled — no versions left
   └─HTTP [cd3eb016] log:
     ├─possible versions are: [0.6.10-0.6.14, 0.7.0-0.7.1, 0.8.0-0.8.19, 0.9.0-0.9.5] or uninstalled
     └─restricted to versions 0.9 by ShortCodes [f62ebe17], leaving only versions 0.9.0-0.9.5
       └─ShortCodes [f62ebe17] log:
         ├─possible versions are: 0.3.2 or uninstalled
         └─ShortCodes [f62ebe17] is fixed to version 0.3.2
```

**Comment by hellemo** [comment]
Thanks for trying out ShortCodes!

It looks like your version of Plotly is holding back the HTTP update. I'm not using Plotly, but it looks like the last release was back in 2018.

I just checked in a temporary environment that I can install ShortCodes.jl in v1.5.4, you could try that first, perhaps;

```
] activate --temp
add ShortCodes
``` 

Perhaps restricting to v0.9 of HTTP.jl is a bit restrictive on my end, but it's the only version I've actually tested.

**Comment by VivaldoMendes** [comment]
@hellemo Thanks for the quick reply. I don't make much use of Plotly but sometimes I need to use it, so ... Anyway, I `rm` Plotly and tried to install ShortCodes, but I got a series of incompatabilities (please see below). 

Your package is a cracker, so terribly useful. I will install v1.6.1 and see what happens.
```
ERROR: Unsatisfiable requirements detected for package HTTP [cd3eb016]:
 HTTP [cd3eb016] log:
 ├─possible versions are: [0.6.10-0.6.14, 0.7.0-0.7.1, 0.8.0-0.8.19, 0.9.0-0.9.5] or uninstalled        
 ├─restricted by compatibility requirements with CSVFiles [5d742f6a] to versions: [0.6.10-0.6.14, 0.7.0-0.7.1, 0.8.0-0.8.19]
 │ └─CSVFiles [5d742f6a] log:
 │   ├─possible versions are: [0.9.0-0.9.1, 0.10.0, 0.11.0, 0.12.0, 0.13.0, 0.14.0, 0.15.0, 0.16.0-0.16.1, 1.0.0] or uninstalled
 │   └─restricted to versions * by an explicit requirement, leaving only versions [0.9.0-0.9.1, 0.10.0, 
0.11.0, 0.12.0, 0.13.0, 0.14.0, 0.15.0, 0.16.0-0.16.1, 1.0.0]
 ├─restricted by compatibility requirements with MarketData [945b72a4] to versions: [0.6.12-0.6.14, 0.7.0-0.7.1, 0.8.0-0.8.19, 0.9.0-0.9.5], leaving only versions: [0.6.12-0.6.14, 0.7.0-0.7.1, 0.8.0-0.8.19]  
 │ └─MarketData [945b72a4] log:
 │   ├─possible versions are: [0.7.1, 0.8.0, 0.9.0, 0.10.0-0.10.1, 0.11.0, 0.12.0-0.12.1, 0.13.0-0.13.6] or uninstalled
 │   └─restricted to versions * by an explicit requirement, leaving only versions [0.7.1, 0.8.0, 0.9.0, 
0.10.0-0.10.1, 0.11.0, 0.12.0-0.12.1, 0.13.0-0.13.6]
 ├─restricted by compatibility requirements with SDDP [f4570300] to versions: 0.8.1-0.8.19
 │ └─SDDP [f4570300] log:
 │   ├─possible versions are: [0.1.0, 0.2.0-0.2.4, 0.3.0-0.3.14] or uninstalled
 │   ├─restricted to versions * by an explicit requirement, leaving only versions [0.1.0, 0.2.0-0.2.4, 0.3.0-0.3.14]
 │   └─restricted by compatibility requirements with RecipesBase [3cdcf5f2] to versions: 0.3.2-0.3.14 or uninstalled, leaving only versions: 0.3.2-0.3.14
 │     └─RecipesBase [3cdcf5f2] log:
 │       ├─possible versions are: [0.4.0, 0.5.0, 0.6.0, 0.7.0, 0.8.0, 1.0.0-1.0.2, 1.1.0-1.1.1] or uninstalled
 │       └─restricted to versions 1 by Plots [91a5bcdd], leaving only versions [1.0.0-1.0.2, 1.1.0-1.1.1]
 │         └─Plots [91a5bcdd] log:
 │           ├─possible versions are: 1.13.0 or uninstalled
 │           └─Plots [91a5bcdd] is fixed to version 1.13.0
 └─restricted by compatibility requirements with ShortCodes [f62ebe17] to versions: 0.9.0-0.9.5 — no versions left
   └─ShortCodes [f62ebe17] log:
     ├─possible versions are: [0.2.3, 0.3.0-0.3.2] or uninstalled
     └─restricted to versions * by an explicit requirement, leaving only versions [0.2.3, 0.3.0-0.3.2]  

```

**Comment by hellemo** [comment]
Just tried with Plotly, and the latest Plotly (0.4.1) seems to be compatible with ShortCodes 0.3.3, so closing this issue.
